motoman_gp110_support/test/launch_test.xml

```diff
@@ -0,0 +1,51 @@
+<!--
```

Comment:
please rename this file to include the variant name (ie `launch_test_gp110.xml`).